bigbluebutton-html5/imports/ui/components/dropdown/list/component.jsx

```diff
@@ -8,16 +8,20 @@ import ListSeparator from './separator/component';
 import ListTitle from './title/component';
 
 const propTypes = {
-  children: PropTypes.arrayOf((propValue, key, componentName, location, propFullName) => {
-    if (propValue[key].type !== ListItem &&
-        propValue[key].type !== ListSeparator &&
-        propValue[key].type !== ListTitle) {
-      return new Error(
-        `Invalid prop \`${propFullName}\` supplied to` +
-        ` \`${componentName}\`. Validation failed.`,
-      );
-    }
-  }),
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf((propValue, key, componentName, location, propFullName) => {
+      if (propValue[key].type !== ListItem &&
+          propValue[key].type !== ListSeparator &&
+          propValue[key].type !== ListTitle) {
+        return new Error(
+          `Invalid prop \`${propFullName}\` supplied to` +
```

Comment:
Is the + really needed here?

Reply:
if this was all on one line it'd be 104 characters (including whitespace) which is breaking the linter rule from airbnb about limiting lines to 100 characters (including whitespace) which is why i would think it was broken up.

i actually just noticed this other interesting airbnb rule that strings seem to be the exception and its okay to have >100 chars if its a string because breaking it up makes it messy, hard to search, etc

https://github.com/airbnb/javascript#strings--line-length

Reply:
I think eslint is capable of making decisions regarding those "exceptions", I think I had a few long lines in #4345 and it didn't complain.
Like [here](https://github.com/bigbluebutton/bigbluebutton/pull/4345/files#diff-89503993ca5e3c87aa355fca9298afdaR12) or [here](https://github.com/bigbluebutton/bigbluebutton/pull/4345/files#diff-e529b94d29c4776e67a304f54fb541ceR4) for example.